charts/common/templates/addons/code-server/_container.tpl

```diff
@@ -24,13 +24,12 @@ ports:
   containerPort: {{ .Values.addons.codeserver.service.port.port }}
   protocol: TCP
 args:
+{{- range .Values.addons.codeserver.args }}
+- {{ . | quote }}
+{{- end }}
 - "--port"
 - "{{ .Values.addons.codeserver.service.port.port }}"
-- "--auth"
-- "none"
-{{- with .Values.addons.codeserver.args }}
-{{ . }}
-{{- end }}
+- {{ .Values.addons.codeserver.workingDir | default (first .Values.addons.codeserver.volumeMounts).mountPath }}
 - {{ .Values.addons.codeserver.workingDir | default (first .Values.addons.codeserver.volumeMounts).mountPath }}
```

Comment:
Is this line supposed to be duplicated?

Reply:
no; fixed in https://github.com/k8s-at-home/charts/pull/414/commits/52829d78616fb2c5b596e41cb16e6fd81f677e6b

Reply:
just saw that :)